lldb/source/Target/Statistics.cpp

```diff
@@ -513,6 +518,63 @@ llvm::json::Value DebuggerStats::ReportStatistics(
   return std::move(global_stats);
 }
 
+void TargetStats::SetFirstBtTime(lldb::ProcessSP process_sp, Thread &thread) {
```

Comment:
No need to pass a process into this function, the thread has access to its process if it is needed.